doc/en/fcron.8.sgml

```diff
@@ -121,18 +121,23 @@ compiled to run in foreground as default.</para>
 		<term><option>--maxserial</option>
 <replaceable>num</replaceable></term>
 		<listitem>
-		  <para>Set to <replaceable>num</replaceable> the maximum number
-		     of serial jobs which can run simultaneously. By default,
-		     this value is set to &serialmaxrunning;.</para>
+		  <para>Run at most <replaceable>num</replaceable> serial jobs
+		     simultaneously. By default, this value is set to &serialmaxrunning;.</para>
 		    <para>&seealso; option &optserial; in &fcrontab;(5).</para>
 		</listitem>
 	    </varlistentry>
 	    <varlistentry>
 		<term><option>-q</option> <replaceable>num</replaceable></term>
 		<term><option>--queuelen</option> <replaceable>num</replaceable></term>
 		<listitem>
-		    <para>Set to n the number of jobs the serial queue and
-the lavg queue can contain.</para>
+		    <para>Sets the size of the queue that is used to manage serial
```

Comment:
I would mention the lavg queue on the first line, e.g. (without the docbook formatting):
```
Sets the size of the queue that is used to manage serial jobs and the lavg queue each to <replaceable>num</replaceable>.

This means that:
- at most <replaceable>num</replaceable> serial jobs can be
			in the waiting state.
- at most <replaceable>num</replaceable> jobs can wait until the system average
			load drops below a certain value.
```

WDYT?

Reply:
good idea.  Will not use more listing/indentation in highlighting, as it gets too crowded then,  but just use A/B in text.

Reply:
Thanks. Looking at it again, with this rewording, I think it should be "sizes of the queues that are used" (plural)?